### ISSUE TITLE
xray fails in second execution (when the first execution is aborted)

### ISSUE THREAD

**Issue by akiradeveloper** [docker-slim/docker-slim#218]
Expected Behavior
=================

xray should complete

---
Actual Behavior
=================

xray fails execution.

---
Steps to Reproduce the Problem
===============================

  1. Run xray a docker image 
  1. Kill the process during xray processing
  1. Rerun xray 

I reproduced this behavior for two different images and they differs in the errors. It seems to me that the xray tries to grab the old garbage cache (or log?) in the previous execution. Maybe, they aren't written to disk safely and this is why such massage is seen.

```
time="2021-09-01T07:20:24Z" level=error msg="layerFromStream: error reading layer(ee708c722ed760eba5eebbbc399b74c048848cf12d47da5a91e9e06db3398bb7) - unexpected EOF"
```

---
Specifications
=================
  - Version: docker-slim version linux|Transformer|1.36.3|243c328df23a2b085238c2e35df8125b366a658c|2021-08-30_07:26:30PM
  - Platform: Ubuntu 4.15.0-122-generic

**Comment by kcq** [comment]
Thank you for the issue @akiradeveloper ! Do you mind checking what you have in the state directory for that xray execution? The state data should be under `/tmp/docker-slim-state/.docker-slim-state` if you installed the docker-slim executable to one of the standard bin directories (otherwise it's in `.dockers-slim-state` in the same directory where you have the main `docker-slim` executable). The image data should be in `images/YOUR_IMAGE_DIGES.T/image`. Do you mind doing an `ls -lh` there?

**Comment by kcq** [comment]
this enhancement ( https://github.com/docker-slim/docker-slim/commit/62e140d86ec90407ca9819fa402b4289f035acb6 ) should address the problem and it'll be available in the next release

**Comment by kcq** [comment]
this should be addressed in the 1.36.4 release: https://github.com/docker-slim/docker-slim/releases/tag/1.36.4
